### ISSUE TITLE
v1.0 misses results

### ISSUE THREAD

**Issue by sbrocket** [atg/Ingredients#18]
I'm not sure how far this bug potentially goes. I verified that this was the case with at least one other person, so I don't think this is something specific to my Xcode install.

If I search for "rootViewController", the only two results I get are -[UINavigationController popToRootViewControllerAnimated:] and the rootViewController property of UIWindow. -[UINavigationController initWithRootViewController:] is not returned as a result even though it clearly should be. This seems like a pretty basic but potentially far reaching issue.

**Comment by atg** [comment]
Indeed. It would seem `initWithRootViewController:` is not being included in the initial index for some reason.
